lualib/rspamadm/mime.lua

```diff
@@ -381,9 +381,9 @@ local function extract_handler(opts)
     end
 
     if opts.words then
-      local howw = opts['words_format'] or 'stem'
+      local how = opts['words_format'] or 'stem'
       table.insert(out_elts[fname], 'meta_words: ' ..
-          print_words(task:get_meta_words(howw), howw == 'full'))
+          print_words(task:get_meta_words(how), how == 'full'))
```

Comment:
It's a local, so I think this is fine (and it matches a later instance of `how`)